core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -221,7 +221,6 @@ object Client {
     val conf = new SparkConf()
     val driverArgs = new ClientArguments(args)
 
-    conf.set("spark.rpc.askTimeout", "10")
```

Comment:
I see, does that mean that the default should be 10s? the real issue is just that the default ends up being "10" instead of advertised 120s.